cookbook/deployment/index.rst

```diff
@@ -8,3 +8,4 @@ Deployment
     azure-website
     heroku
     platformsh
+    fortrabbit
```

Comment:
We also need to add an entry to `/cookbook/map.rst.inc` to make the article appear on the index too.